ptsemseg/models/pspnet.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, feature_scale=4, n_classes=21, is_deconv=True, in_channels=3,
         self.finaldeconvbnrelu1 = nn.Sequential(nn.ConvTranspose2d(filters[0], 32/feature_scale, 3, 2, 1),
                                       nn.BatchNorm2d(32/feature_scale),
                                       nn.ReLU(inplace=True),)
-        self.finalconvbnrelu2 = conv2DBatchNormRelu(32/feature_scale, 3, 32/feature_scale, 1, 1)
+        self.finalconvbnrelu2 = conv2DBatchNormRelu(in_channels=32/feature_scale, k_size=3, n_filters=32/feature_scale, padding=1, stride=1)
```

Comment:
Yes, while working on the model, I realized that this is just a resnet encoder decoder, the parsing module seems missing. (I guess you are in the middle of finishing the code :) ) I am implementing few more things on top of your code, because it already has quite a few things that I would need to write fully functioning pspnet. If you do not mind I will create some more pull request this week. Thanks again!